web/themes/custom/customtheme/templates/block/block--customtheme-customweatherblock.html.twig

```diff
@@ -25,7 +25,9 @@
  * @see template_preprocess_block()
  */
 #}
- {% set weather_text = content['#weather_text'] %}
+{% set city = content['#selected_city'] %}
```

Comment:
You can use these changes anyway, without the content['*']

Reply:
I can't. This is the only possible way.